cypress/support/datasetImportPage.js

```diff
@@ -41,9 +41,7 @@ export const importMoreDataset = (filename, mimeType = 'text/csv') => {
     cy.get('#confirm-upload', { timeout: 1000 }).should('be.visible');
     cy.wait(300);
     cy.contains('Accept').click({ force: true });
-    cy.get('.progress').should('exist');
-    cy.get('.progress', { timeout: 6000 }).should('not.exist');
-
+    cy.wait(1000);
```

Comment:
I don't know about you, but Julien told me that it was better to make a get with a timeout than to wait.

`cy.get('.data-published a', { timeout: 1000 }).contains`

Reply:
Yes, because it returns earlier if the element is found and hence speeds up tests

Reply:
godd advice, I change it